packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -111,10 +112,8 @@ export type CreateViewResult = {
     view: VideoStreamRendererView;
 };
 
-// @public
-export type DeclarativeCallAgent = CallAgent & /* @conditional-compile-remove(one-to-n-calling) */ {
-    incomingCalls: ReadonlyArray<DeclarativeIncomingCall>;
-};
+// @beta
+export type DeclarativeCallAgent = CallAgent & IncomingCallManagement;
```

Comment:
Rename to `StatefulCallAgent`. Not sure whether this PR added it or just an artifact of needing to merge from `main`.

Reply:
![image](https://user-images.githubusercontent.com/11863655/199816648-d5d1e627-f7f1-4150-9228-ef4da2227ea5.png)
Looks like it is DeclarativeCallAgent name in the main (I only extra the IncomingCallManagement ckass, but not renaming)
I can create a PR to rename this if we want (or we have a PR resolving this but not merged?)